dist/yall.js

```diff
@@ -138,6 +139,12 @@ function yall (options) {
 
     if (observeChanges) {
       yallApplyFn(queryDOM(options.observeRootSelector || "body"), yallCreateMutationObserver);
+    } else {
+    // IntersectionObserver not supported
+      if (noPolyfill) {
+				yallApplyFn(lazyElements, yallBindEvents);
```

Comment:
Rather than needing to change how events work, we could just bind them before doing the lazy-load. I know that at my company we bind a load event do do some cleanup, so it would be nice to get the same event for all browsers, regardless of if the image is actually lazy loaded.

Reply:
@malchata what are your thoughts on this approach?

Reply:
I apologize, @radfahrer. The holidays have me busier than ever and I haven't had a chance to pull this down and test it. I'll give it a shot soon. Thanks for being so patient. 🙏

Reply:
You are right. I don't know why I put it here. Fix incoming.